docs/rfcs/0000-rfc-template.md

```diff
@@ -0,0 +1,126 @@
+# RFC-NNNN: Human-readable title goes here
+
+Remember to add autogenerated HackMD badge here. See [README.md](README.md) for more details.
+
+**Authors** (in alphabetical order):
+
+- Author Name, [@Author_GitHub_Handle](https://github.com/Author_GitHub_Handle)
+
+**Status** (as defined [here]): `Provisional`
+
+[here]: https://github.com/kubernetes/enhancements/blob/master/keps/0001-kubernetes-enhancement-proposal-process.md#kep-metadata
+
+**Creation Date**: `YYYY-MM-DD`
+
+**Last Updated**: `YYYY-MM-DD`
+
+**RFC Handle**: `rfc-template` (should match the file name, as `NNNN-{rfc_handle}`)
+
+**Initial Pull Request**: [racklet/racklet#NNNN](https://github.com/racklet/racklet/pull/NNNN)
+
+**Tracking Issue**: [racklet/racklet#NNNN](https://github.com/racklet/racklet/issues/NNNN)
+
+[TOC]
```

Comment:
Do we have a guideline for how to format the table of contents? Is there a feature in HackMD for automatically generating this?

Reply:
Both HackMD and the `mdbook-toc` plugin generates this automatically, I'll add a note about that.